packages/tocco-ui/src/utilStyles/declareInteractionColors.spec.js

```diff
@@ -169,7 +108,75 @@ describe('tocco-ui', () => {
 
         scenarios.map(scenario => {
           const {color, expectation, options} = scenario
-          expect(generateInteractionColor(color, options)).to.deep.equal(expectation)
+          expect(generateShades(color, options)).to.deep.equal(expectation)
+        })
+      })
+
+      test('should generate interaction color', () => {
+        const scenarios = [{
+          scheme: 'flatBase',
+          expectation: {
+            bg: ['#300', '#600', '#900'],
+            fg: ['#900', '#c00', '#f00']
+          }
+        }, {
+          scheme: 'flatPrimary',
+          expectation: {
+            bg: ['#300', '#600', '#900'],
+            fg: ['#600', '#900', '#c00']
+          }
+        }, {
+          scheme: 'raisedBase',
+          expectation: {
+            bg: ['#600', '#900', '#c00'],
+            fg: ['#900', '#c00', '#f00']
+          }
+        }, {
+          scheme: 'raisedPrimary',
+          expectation: {
+            bg: ['#600', '#900', '#c00'],
+            fg: ['#900', '#c00', '#f00']
+          }
+        }, {
+          scheme: {
+            bg: 'colors.primary',
```

Comment:
IMO it would be more intuitive to prefix everything that's not a color?
`fg: '#234, $colors.text, black'`

what do you think? You could remove dependency and just check for the $

Reply:
Using `color.startsWith('colors.')` instead of prefixing theme paths for colors by $ prevents to introduce a special case for a few color paths.